### PR TITLE
Switch MacOS to x86_64

### PR DESCRIPTION
**Pull request by SConaway**

### CODE REVIEW

File: package-avrdude.bash [hunk 1]
```diff
@@ -93,7 +93,6 @@ else
   rm -f avrdude-${OUTPUT_VERSION}-${OUTPUT_TAG}.tar.bz2
   cp -a objdir avrdude
   tar -cjvf avrdude-${OUTPUT_VERSION}-${OUTPUT_TAG}.tar.bz2 avrdude
-  rm -r avrdude
+#  rm -r avrdude
```
Comment (per1234) [inline package-avrdude.bash]: Was this change intentional? It doesn't seem related to the stated purpose of the PR.

Comment (SConaway) [reply package-avrdude.bash]: Nope.

Comment (SConaway) [reply package-avrdude.bash]: Fixed now.

**Comment by SConaway** [comment]
I can't test this build on my mac because of the missing symbols, but it seems like it'll work.

**Comment by SConaway** [comment]
@per1234 merge?

**Comment by per1234** [comment]
@SConaway I'm not able to make decisions on whether the PR will be merged. My approval was only meant to indicate that the issue I brought up in my review has been resolved. After I did it, I realized that it could have been interpreted as approval of the PR as a whole, and I don't have enough knowledge on the subject to make that call. I still haven't completely figured out how the GitHub review system works as I only recently started using it.

**Comment by SConaway** [comment]
Sounds good.

Steven

**Comment by facchinm** [comment]
@SConaway I'd prefer to merge it on `staging` branch (which is the one we are actually use). Are you ok with that?

**Comment by SConaway** [comment]
Yep.

**Comment by SConaway** [comment]
I guess this should be closed? Can we push a build at x86_64?

**Comment by facchinm** [comment]
Yes, it can be closed, `avrdude-6.3.0-arduino17` ships with 64 bit build for osx
